tests/cut-plane/test_all.py

```diff
@@ -40,13 +33,13 @@ def test_cut_neuron_simple():
 def test_cut_real_neuron():
     filename = DATA / 'Neuron_slice.h5'
     result = CutPlane.find(filename, bin_width=10).to_json()
-    assert_equal(result['status'], 'ok')
+    assert result['status'] == 'ok'
     assert_almost_equal(result['cut-plane']['a'], 0)
```

Comment:
I am wondering what is the best in such case:
`assert_almost_equal(result['cut-plane']['a'], 0)`
or
`assert result['cut-plane']['a'] == pytest.approx(0)`
?

Reply:
I don't know. Lets keep as is for now.